src/presets/cloudflare.ts

```diff
@@ -19,6 +19,7 @@ const cloudflareNodeCompatModules = [
   "stream/promises",
   "stream/web",
   "string_decoder",
+  "util/types",
```

Comment:
Just checking, do you have any upstream test that makes sure `util/types` will support future additions from Node?

Reply:
Workerd doesn't run Node.js in the test suite. The missing functionality will be tracked with @jasnell and mine best efforts. FYI: it's highly unlikely to have any new methods in util/types.